docs/getting-started-with-scalardb.md

```diff
@@ -6,36 +6,34 @@ If you haven't done it, please configure them first by following [this](getting-
 ## Build
```

Comment:
Deleted the Storage API thing in Getting Started because users usually don't use Storage API. Users usually use Transaction API only.

However, we have some use cases for Storage API mainly for performance reasons. So, I will add another document for Storage API (maybe for experts) in another PR.